### PR TITLE
fix: adding extraConfig to worker deployment #20828

### LINKED ISSUE #20828: volumeMounts for extraConfigs are missing in the deployment-worker.yaml
Hi,

we try to run superset on kubernetes with your helm charts and a custom security manager. we use the extraConfigs to mount some config files needed by flask_oidc (client_secret.json) - in the superset node, everything is fine, but the worker node crashes as the worker also initializes the flask_oidc context and does not mount the extraConfigs

working deployment for the superset node

https://github.com/apache/superset/blob/master/helm/superset/templates/deployment.yaml#L111

missing extraConfigs in the worker node :cry: 

https://github.com/apache/superset/blob/master/helm/superset/templates/deployment-worker.yaml#L104

### Expected results

the worker node mounts all the extraConfigs under the extraVolumeMounts like the main node

### Actual results

the worker node crashes, as the extraConfigs were not mounted

### PR DESCRIPTION
**Pull request by cdreier**
Fixes #20828

### CODE REVIEW

File: helm/superset/templates/deployment-worker.yaml [hunk 1]
```diff
@@ -130,6 +135,10 @@ spec:
         - name: superset-config
           secret:
             secretName: {{ tpl .Values.configFromSecret . }}
-      {{- with .Values.extraVolumes }}
-      {{- tpl (toYaml .) $ | nindent 8 -}}
-      {{- end }}
+        - name: superset-extra-config
```
Comment (craig-rueda) [inline helm/superset/templates/deployment-worker.yaml:138]: Looks like this is missing a leading `{{- if .Values.extraConfigs }}`

**Comment by craig-rueda** [comment]
@cdreier - can you pls bump the chart version?

File: helm/superset/templates/deployment-worker.yaml (hunk 1 above)
Comment (cdreier) [reply helm/superset/templates/deployment-worker.yaml:139]: good catch, i just missed it :+1:

**Comment by cdreier** [comment]
thanks a lot for your support and fast response!

**Comment by cdreier** [comment]
@craig-rueda is there a chance we get this merged this week? would help us a lot to remove temporary workarounds - or is anything missing?

**Review by craig-rueda — APPROVED** [review]

**Pull request merged**